src/particle/particle/convert.py

```diff
@@ -75,13 +85,24 @@
     Inv_mapping,
     Parity,
     Parity_mapping,
-    SpinType,
-    Status,
```

Comment:
These were unused. Assuming users should not do `particle.particle.SpinType`, but either `particle.SpinType` or `particle.enums.SpinType`.

Reply:
Safe to remove for this script. In fact I removed a few such unused imports also in PR https://github.com/scikit-hep/particle/pull/324 but did not run a comprehensive check. Good that you did it 👍.